src/common/hook/useQueue/utils/publish.tsx

```diff
@@ -1,9 +1,13 @@
-import { wrapDocuments } from "@govtechsg/open-attestation";
+import {
+  wrapDocuments,
+  __unsafe__use__it__at__your__own__risks__wrapDocuments,
+  utils,
```

Comment:
```suggestion
  wrapDocuments,
  __unsafe__use__it__at__your__own__risks__wrapDocuments as wrapDocumentsV3,
  utils,
```

maybe can alias so can know that's v3 wrap

Reply:
done.